desc/plotting.py

```diff
@@ -1889,17 +2104,29 @@ def plot_boozer_modes(
                 linestyle=linestyle,
                 linewidth=linewidth,
             )
+        plot_data[f"|B|_{M}{N}"] = B_mn[:, i]
```

Comment:
probably cleaner to return `B_mn` as an array and then a separate array for `B modes`